test/sqs_lambda_to_batch_fargate/lambda/export_logs.py

```diff
@@ -0,0 +1,73 @@
+import json
+import logging
+import os
+import random
+import string
+
+import boto3
+
+s3_client = boto3.client("s3")
+s3_resource = boto3.resource("s3")
+logs_client = boto3.client("logs")
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+log_group_name = "/aws/batch/job"
+
+
+def handler(event, context):
+    logger.debug(f"BUCKET_NAME: {os.environ['BUCKET_NAME']}")
+    logger.debug(f"INCOMING EVENT: {event}")
+
+    if "Batch Job State Change" not in event["detail-type"]:
+        logger.info(f"Non-triggering Batch event: {event['detail-type']}")
+        return
+    if "TIMED_OUT" in event["detail"]["status"]:
+        raise Exception(
+            "Job timed out. Contact application owner or increase time out threshold"
+        )
+    if not (
+        "FAILED" in event["detail"]["status"]
```

Comment:
Isn't "FAILED" the complete status string? I think you could do this as

`if event["detail"]["status"] not in ["FAILED", "SUCCEEDED"]:`

Reply:
Yeah that actually works fine.